package.json

```diff
@@ -64,14 +63,14 @@
     "test:only": "nyc --reporter=html --reporter=text-summary npm run mocha:test",
     "test:runtime": "TZ=America/Los_Angeles wdio wdio.conf.js",
     "test:runtime:generate": "rm -Rf test-runtime/resources && VL_GENERATE_TESTS=true npm run test:runtime",
-    "test:debug": "npm run tsc && mocha --recursive --debug-brk --inspect build/test",
-    "test:debug-examples": "npm run tsc && npm run schema && mocha --recursive --debug-brk --inspect build/examples",
+    "test:debug": "tsc && mocha --recursive --debug-brk --inspect build/test",
+    "test:debug-examples": "tsc && npm run schema && mocha --recursive --debug-brk --inspect build/examples",
     "mocha:test": "mocha --require source-map-support/register --reporter dot --recursive build/test",
     "mocha:examples": "mocha --require source-map-support/register --reporter dot --recursive build/examples",
 
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
     "watch:build": "watchify src/index.ts -p tsify -v -d -s vl -o 'exorcist build/vega-lite.js.map > build/vega-lite.js'",
-    "watch:tsc": "npm run tsc -- -w",
+    "watch:tsc": "tsc -- -w",
```

Comment:
do you still need `--` here?

Reply:
Nope. It's not needed anymore.

Reply:
Should I just delete this entry then? You can just call `yarn tsc -w`.

Reply:
I never use it so fine with me.